spec/system/reward_buying_spec.rb

```diff
@@ -40,5 +40,24 @@
       expect(page).to have_content 'Reward bought'
       expect(page).to have_content 'Received Points: 0'
     end
+
+    it 'checks pagination' do
```

Comment:
it should be names something like 'it paginates rewards', also not sure, if it should be in the 'when we want to buy reward' context

also it would be good to check, that each page displays REWARDS_PER_PAGE rewards per page